packages/localizations/src/en-US.ts

```diff
@@ -576,6 +576,8 @@ export const enUS: LocalizationResource = {
     form_identifier_exists: '',
     form_password_validation_failed: 'Incorrect Password',
     form_password_not_strong_enough: 'Your password is not strong enough.',
+    form_password_size_in_bytes_exceeded:
+      'Your password has exceeded the maximum number of bytes allowed, please shorten it or remove some special characters.',
```

Comment:
❓I understand that fapi will have to throw an error in that case, but why does the end user need to know about "bytes" ? The term feels quite technical for someone who just want to sign up. 

Should we try something like "Invalid password, please shorten it or remove some special characters."

Reply:
@panteliselef yes it seems odd

> Why is there a change in `package-lock.json`? Seems unrelated.
> 
> Other than that, looks good to me.

This resulted after pulling the latest main branch and doing `npm install`.  cc @nikosdouvlis do you think this is an issue?